org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/ZkReportViewer.java

```diff
@@ -1458,6 +1494,28 @@ private void cmd_report()
 		postRenderReportEvent();
 	}	//	cmd_report
 
+	/**
+	 * Refresh With Parameters
+	 */
+	private void cmd_reRun() {
+		int AD_Process_ID = m_reportEngine.getPrintInfo() != null ? m_reportEngine.getPrintInfo().getAD_Process_ID() : 0;
+		ProcessInfo pi = new ProcessInfo("RefreshWithParameters", AD_Process_ID);

```

Comment:
can AD_Process_ID be 0 here ? if yes, then the method should stop immediately (i.e return;) if AD_Process_ID is 0